_includes/tools/mocha/usage.md

```diff
@@ -1,9 +1,15 @@
+#### Mocha 4
+
+Updated in Mocha v4.0.0
```

Comment:
Could you maybe remove the first sentence, it's redundant with the rest to me.

Reply:
Thanks, done